dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -69,8 +69,8 @@
         - data['answers'].each_with_index do |answer, i|
           -# When LevelGroup-hosted, with students (or teachers doing PD), set all answers false before rendering page.
           - correct = include_multi_answers?(standalone) ? answer['correct'] : false
-          - answer_class = "answerbutton " + (use_tight_layout ? "" : "btn")
-          %div{class: answer_class, style: "display: inline-block; height: #{height}", correct: "#{correct}", index: "#{i}"}
+          - answer_class = "answerbutton " + (use_tight_layout ? "tight_layout" : "btn")
```

Comment:
```suggestion
          - answer_class = "answerbutton " + (use_tight_layout ? "tight-layout" : "btn")
```
The `_` needs to be a `-`. Otherwise the name doesn't match the CSS class.